core/funds.py

```diff
@@ -31,3 +30,20 @@ def search(to_search: dict, word_list: List[str]):
                 break
 
     return return_list
+
+def get_funds(search_items = None):
+    list_of_funds = fund_data.FUNDS_DUMMY_DAO.get_all()
+    if search_items is None:
+        if type(list_of_funds) == list:
```

Comment:
🤔 What else could "list_of_funds" be?

Reply:
💯 renamed to 'funds_list' which hopefully is less verbose 😉